.github/workflows/runner.yml

```diff
@@ -96,15 +96,25 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v3
 
+    - name: Install Ansible
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install ansible
+
     - name: Build container
       run: |
-        docker buildx build -f ./share/containers/${{ matrix.os }}.dockerfile . --output build-out
```

Comment:
BTW, what if we would map some volume to write the log files to it with docker(1)?

```docker
RUN bash -c "trap 'cat <tests/unit/test-suite.log | tee /some/mapped/volume/test-suite.log >&2' ERR; make check;"
```

Reply:
That would allow storing the artifacts without needing ansible.

Reply:
You can't mount volumes while building a container image, and that's exactly what we are doing with dockerfile. I have tried to solve this problem in various ways and with the technology we use it is not possible :sweat:

Reply:
How about something like this?

```sh
docker build ... | tee >(sed -n '/BEGIN MARKER/,/END MARKER/p' >/host/log/file)
```

We would only need to find some consistent markers in the log.

Reply:
Same answer as in https://github.com/shadow-maint/shadow/pull/1009#discussion_r1638276746

Reply:
If each log file has a different marker, you can parse each one separately:

```sh
docker build ... | tee >(sed -n '/BEGIN A/,/END A/p' >/host/A.log) >(sed -n '/BEGIN B/,/END B/p' >/host/B.log)
```